hws/src/dr_definer.py

```diff
@@ -172,8 +172,8 @@ def parse_data(self):
             _dw_fields["dw_selector_base_1"] = dr_hl_fields_arr_add_prefix("base_1: ", self.dw_fields["dw_selector_3"])
             _dw_fields["dw_selector_arg_0"] = dr_hl_fields_arr_add_prefix("arg_0: ", self.dw_fields["dw_selector_4"])
             _dw_fields["dw_selector_arg_1"] = dr_hl_fields_arr_add_prefix("arg_1: ", self.dw_fields["dw_selector_5"])
-            self.compare_dw_fields = self.dw_fields
-            self.dw_fields = _dw_fields
+            self.compare_dw_fields = _dw_fields
+            #self.dw_fields = _dw_fields
```

Comment:
do we need this commented out line here?

Reply:
Fixed